libraries/app/include/graphene/app/database_api.hpp

```diff
@@ -663,9 +663,12 @@ class database_api
       bool           verify_authority( const signed_transaction& trx )const;
 
       /**
-       * @return true if the signers have enough authority to authorize an account
+       * @brief Verify that the public keys have enough authority to approve an operation
```

Comment:
I think we should mention the account in this sentence and the return

Reply:
I think the latest commit will be more to your liking. Please take a look.